app/models/fax.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Fax
+  delegate \
+    :sfax_api_key,
+    :sfax_encryption_key,
+    :sfax_init_vector,
+    :sfax_username,
+    to: :"Rails.application.secrets"
+
+  def self.send_fax(number:, file:)
+    new(number, file).send_fax
+  end
+
+  def initialize(number, file)
+    @number = number
+    @file = file
+  end
+
+  def send_fax
+    sfax.send_fax(number, file)
+  end
+
+  protected
```

Comment:
It's mostly out of habit at this point but the reason is generally for purposes like equality checking. A protected attribute allows an instance of a class to inquire about another instance's values ... which, for things like value objects is a necessity.

Instead of thinking about it, I've just gotten used to putting `attr_*` in the protected object space.